### DISCUSSION TITLE
浮窗图标 不显示系统 mac10.15.7  正常吗

### DISCUSSION THREAD

**Discussion by dishengai** [mushan0x0/AI0x0.com#350] [Q&A]
浮窗图标 不显示系统 mac10.15.7  正常吗

**Comment by mushan0x0** [comment]
<img width="206" alt="image" src="https://github.com/mushan0x0/AI0x0.com/assets/29084441/37e7c28b-edb0-4da5-8959-217d83b06f7f">
这个不显示吗

**Comment by dishengai** [comment]
是的 不显示这个,是系统太老了不支持吗

**Comment by mushan0x0** [comment]
应该是的
